scripts/partition_manager/partition_manager.rst

```diff
@@ -717,11 +717,14 @@ The sizes of each partition are determined by the associated :file:`pm.yml` file
 Static configuration
 ********************
 
+The Partition Manager provides static configuration that lets you define static partitions.
+This feature can be used with single-image applications, but its most common use case is with multi-image ones.
```

Comment:
The static configuration can be used in general when Partition Manager manages memory partitions. I would not go deeper into details here (e.g. Partition Manager for single-image applications requires additional configuration and it's not used by default).